lib/cartopy/tests/test_util.py

```diff
@@ -64,3 +65,322 @@ def test_invalid_coord_size(self):
     def test_invalid_axis(self):
         with pytest.raises(ValueError):
             add_cyclic_point(self.data2d, axis=-3)
+
+
+class Test_add_cyclic:
```

Comment:
Ignoring the existing test, no need to break pep8 naming here with the docstring.
```suggestion
class TestAddCyclic:
```